portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,121 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Sherry's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+    <!--Home Initial View-->   
+    <header id="home" class="bg">
+      <div id="nav">
+        <ul>
+          <li><a href="#gallery">Gallery</a></li>
+          <li><a href="#projects">Projects</a></li>
+          <li><a href="#resume">Resume</a></li>
+          <li><a href="#about">About Me</a></li>
+          <li><a href="#home">Home</a></li>
+        </ul>  
+      </div>
+      <div class="abs-center">
+        <h1>Sherry Shi</h1>
+        <p>Pictured: Port Washington Sunset | pc: Google Images</p>
+        <p>Click here to get a random fact about me:</p>
+        <button onclick="genRandomFact()">Fact</button>
+        <p id="fact-container"></p>
+      </div>
+    </header>
+
+    <!-- About Section --> 
+    <section id="about" class="dark">
+      <div class="container">
+        <h3 class="bluec">About Me</h3>
+        <div class="wrapper bio">
+          <img class="propic" src="/images/ryan.jpg" alt = "My Profile Picture">
```

Comment:
propic -> pro-pic
Try to use a hyphen as a delimiter https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
done